python/lex_tools.i

```diff
@@ -0,0 +1,60 @@
+%module lextools
+
+%{
+#define SWIG_FILE_WITH_INIT
+#include <lrx_processor.h>
+
+
+class LRX: public LRXProcessor
+{
+public:
+  /**
+   * Imitates functionality of lrx_proc using file path
+   */
+  void lrx_proc(char arg, char *dictionary_path, char *input_path, char *output_path);
+};
+
+
+void
+LRX::lrx_proc(char arg, char *dictionary_path, char *input_path, char *output_path)
```

Comment:
Is there a reason this can't just be combined with the declaration above?

Reply:
The class declaration is required to for wrapping. By keeping the definition separate, I simply copy paste the declaration, thus minimizing the human error (got stuck at this in past). Hence I prefer to keep the code separate. But this shouldn't be an issue now. Should I move that part in the class declaration?

Reply:
I see what you're saying. However, now you've written it 3 times! The compiler should reject any human error with mismatched interfaces anyway so let's go ahead and combine as possible :)

(On another note, these files are really annoying to read without syntax highlighting. I wonder if we can just tell GH to treat them as C++ - it doesn't support SWIG unfortunately)